core/templates/pages/exploration-player-page/services/exploration-player-state.service.ts

```diff
@@ -53,8 +53,7 @@ angular.module('oppia').factory('ExplorationPlayerStateService', [
   'PlaythroughService', 'PretestQuestionBackendApiService',
   'QuestionBackendApiService', 'QuestionPlayerEngineService',
   'ReadOnlyExplorationBackendApiService', 'StateClassifierMappingService',
-  'StatsReportingService', 'UrlInterpolationService', 'UrlService',
```

Comment:
Hm, then maybe we can remove the require statement for UrlInterpolationService as well.

https://github.com/oppia/oppia/blob/b1ba8d11e81b968cddaaa66180f8b787e7c72372/core/templates/pages/exploration-player-page/services/exploration-player-state.service.ts#L29

Reply:
Done.